requirements.txt

```diff
@@ -35,7 +35,7 @@ python-dateutil==2.8.2 ; python_version >= '2.7' and python_version not in '3.0,
 pyyaml==6.0 ; python_version >= '3.6'
 pyyaml-env-tag==0.1 ; python_version >= '3.6'
 regex==2022.10.31 ; python_version >= '3.6'
-requests==2.31.0 ; python_version >= '3.7' and python_version < '4'
+requests==2.28.2 ; python_version >= '3.7' and python_version < '4'
```

Comment:
Mind reverting? This was part of a renovate security update this morning.

Reply:
Thanks, I didn't include it initially, but I'm trying to figure out why circle is saying there are dist changes...